src/Compilers/CSharp/Portable/FlowAnalysis/DefiniteAssignment.LocalFunctions.cs

```diff
@@ -112,10 +118,10 @@ private void RecordReadInLocalFunction(int slot)
             }
         }
 
-        private BitVector GetCapturedBitmask(ref BitVector state)
+        private BitVector GetCapturedBitmask()
         {
-            BitVector mask = BitVector.Empty;
-            for (int slot = 1; slot < state.Capacity; slot++)
+            BitVector mask = BitVector.AllSet(1);
```

Comment:
>1 [](start = 46, length = 1)

`nextVariableSlot` (so the internal array doesn't have to be reallocated).  Then in the loop,
`mask[slot] = IsCapturedInLocalFunction(slot)`